static/docs/command-reference/checkout.md

```diff
@@ -17,59 +17,54 @@ positional arguments:
-to match with the <abbr>cached</abbr> files that correspond to those checksums.
-
-Using an SCM like Git, the DVC-files are kept under version control. At a given
-branch or tag of the SCM repository, the DVC-files will contain checksums for
-the corresponding data files kept in the cache. After an SCM command like
-`git checkout` is run, the DVC-files will change to the state at the specified
-branch or commit or tag. Afterwards, the `dvc checkout` command is required in
-order to synchronize the data files with the currently checked out DVC-files.
-
-This command must be executed after `git checkout` since Git doesn't track files
-that are under DVC control. For convenience a Git hook is available, simply by
-running `dvc install`, that will automate running `dvc checkout` after
-`git checkout`. See `dvc install` for more information.
-
-The execution of `dvc checkout` does:
-
-- Scan the `outs` entries in DVC-files to compare with the currently checked out
-  data files. The scanned DVC-files is limited by the listed `targets` (if any)
-  on the command line. And if the `--with-deps` option is specified, it scans
-  backward from the given `targets` in the corresponding
-  [pipeline](/doc/command-reference/pipeline).
-
-- For any data files where the checksum doesn't match their DVC-file entry, the
-  data file is restored from the cache. The link strategy used (`reflink`,
-  `hardlink`, `symlink`, or `copy`) depends on the OS and the configured value
-  for `cache.type` – See `dvc config cache`.
-
-Note that this command by default tries NOT to copy files between the cache and
-the workspace, using reflinks instead when supported by the file system. (Refer
-to
+which data files or directories from the <abbr>cache</abbr> should be in use. We
+call these files <abbr>outputs</abbr>, and their checksums are saved in the
+`outs` fields inside DVC-files to achieve this.
+
+When using Git, different DVC-files versioned in separate
+[revisions](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+probably specify different data files from the cache. When switching to those
+versions (with Git commands such as `git checkout`), the current DVC-files will
+no longer match with all of the data in the <abbr>workspace</abbr>.
+
+The `dvc checkout` command synchronizes the workspace data to match with the
+current DVC-files, using a mechanism described below.
+
+💡 For convenience, a Git hook is available to automate running `dvc checkout`
+after `git checkout`. Use `dvc install` to install it.
+
+The execution of `dvc checkout` does the following:
+
+- Scans the `outs` field values in DVC-files to compare with the outputs
+  currently in the <abbr>workspace</abbr>. Scanning is limited to the given
+  `targets` (if any).
+
+- Missing data files or directories, or those with checksums that don't match
+  any DVC-file, are restored from the cache. If the `--relink` option is used,
```

Comment:
if we decided to remove `--with-deps` why do we mention `--relink`? Especially here in the execution procedure. It makes sense to mention it if it's important in a paragraph above? or as a separate paragraph?

Reply:
You're right. I'm removing this sentence about `--relink` here, just moving part of it to the **Options** section. I'll mention all the options in these bullets just briefly instead, so the user knows to scroll down to options to learn about them. See 7deb5e29